lnbits/app.py

```diff
@@ -52,47 +50,25 @@ def create_app(config_object="lnbits.settings") -> FastAPI:
             "url": "https://raw.githubusercontent.com/lnbits/lnbits-legend/main/LICENSE",
         },
     )
+
     app.mount("/static", StaticFiles(packages=[("lnbits", "static")]), name="static")
     app.mount(
         "/core/static",
         StaticFiles(packages=[("lnbits.core", "static")]),
         name="core_static",
     )
 
-    origins = ["*"]
+    # needed for lnurlw?
+    # g().config = settings
```

Comment:
Please remove comments or specify what it refers to. Which line is needed for lnurlw?

Reply:
removed the comment